staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2694,6 +2694,11 @@ type PodSpec struct {
 	// +k8s:conversion-gen=false
 	// +optional
 	HostIPC bool `json:"hostIPC,omitempty" protobuf:"varint,13,opt,name=hostIPC"`
+	// Use the host's user namespace.
+	// Optional: Default to false.
+	// +k8s:conversion-gen=false
+	// +optional
+	HostUserNamespace *bool `json:"hostUserNamespace,omitempty" protobuf:"varint,28,opt,name=hostUserNamespace"`
```

Comment:
similar to ShareProcessNamespace, you may want to reference that the field is alpha and tied to feature gate.

Reply:
@liggit already added at line #2699.